amlip_cpp/test/manual/dds/directwriter.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file directwriter.cpp
+ *
+ */
+
+#include <thread>
+
+#include <ddsrouter_utils/Log.hpp>
+
+#include <types/AmlipIdDataType.hpp>
+#include <dds/Participant.hpp>
+
+int main(
+        int argc,
+        char** argv)
+{
+    // Activate log
+    eprosima::ddsrouter::utils::Log::SetVerbosity(eprosima::ddsrouter::utils::Log::Kind::Info);
+
+    logUser(AMLIP_MANUAL_TEST, "Starting Manual Test DirectWriter execution. Creating Participant...");
+
+    {
+        // Create Participant
+        eprosima::amlip::dds::Participant participant("ManualTestParticipant");
+
+        logUser(AMLIP_MANUAL_TEST, "Created Participant: " << participant << ". Creating Reader for getting Id...");
+
+        // Create Reader to get id of the target
+        eprosima::fastdds::dds::DataReaderQos qos;
+        qos.durability().kind = eprosima::fastdds::dds::DurabilityQosPolicyKind::TRANSIENT_LOCAL_DURABILITY_QOS;
+        qos.reliability().kind = eprosima::fastdds::dds::ReliabilityQosPolicyKind::RELIABLE_RELIABILITY_QOS;
+        qos.history().kind = eprosima::fastdds::dds::HistoryQosPolicyKind::KEEP_ALL_HISTORY_QOS;
```

Comment:
I'm not sure this is the most appropriate choice. Same for `TargetedReader`.

Reply:
It is probably not. I am open to suggestions.